internal/rbd/replicationcontrollerserver.go

```diff
@@ -375,6 +379,7 @@ func (rs *ReplicationServer) DemoteVolume(ctx context.Context,
 // ResyncVolume extracts the RBD volume information from the volumeID, If the
 // image is present, mirroring is enabled and the image is in demoted state.
 // If yes it will resync the image to correct the split-brain.
+// nolint:gocyclo // reduce complexity
```

Comment:
using `FIXME: reduce complexity.` will allow it to also get listed in https://www.tickgit.com/browse?repo=github.com/ceph/ceph-csi so that it is easier to track.